lib/main.js

```diff
@@ -114,7 +114,9 @@ Lambda.prototype._params = function (program, buffer) {
       Variables: {}
     },
     DeadLetterConfig: {
-      TargetArn: ''
+      // Set default value to null.
+      // Because createFunction fails if it is empty character.
+      TargetArn: null
     }
```

Comment:
@DeviaVir 
By writing `AWS_DLQ_TARGET_ARN=` in `.env`, empty characters will pass and you can cancel the setting.